lib/manageiq/environment.rb

```diff
@@ -162,6 +160,18 @@ def self.system!(*args)
       options[:chdir] ||= APP_ROOT
       system(*args, options) || abort("\n== Command #{args} failed in #{options[:chdir]} ==")
     end
+
+    def self.ci?
+      ENV.fetch("CI", nil) == "true"
```

Comment:
GHA sets `CI=true` according to https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables